pkg/health/types.go

```diff
@@ -115,6 +177,19 @@ type Registry struct {
 	// HealthIndicators are the currently registered health indicators
 	HealthIndicators []Indicator
 
-	// HealthAggregationPolicy is the registered health aggregationPolicy
-	HealthAggregationPolicy AggregationPolicy
+	// Indicator Settings of the registry
+	HealthSettings map[string]*IndicatorSettings
+}
+
+// ConfigureIndicators configures registry's indicators with provided settings
+func (r *Registry) ConfigureIndicators() {
```

Comment:
This seems like a hidden extra step that needs to be performed. What do you think about hiding the indicators and add a `func (Registry) AddHealthIndicator(Inidicator)` that configures it?